templates/node/index.d.ts.twig

```diff
@@ -131,11 +131,13 @@ declare module "{{ language.params.npmPackage|caseDash }}" {
 
     static fromBuffer(buffer: Buffer, filename: string): InputFile;
 
-    static fromBlob(blob: Blob, filename: string): InputFile;
+    static fromBlob(blob: buffer.Blob, filename: string): Promise<InputFile>;
```

Comment:
We are adding `Promise` here but not into other functions  (fromStream, fromPlainText, ...). Is that on purpose?

Reply:
Yes, because this is the only async function.